### PR TITLE
synchronize remove unused block

### LINKED ISSUE #63954: Synchronize module show SyntaxError
##### SUMMARY

When there is a file named `*SyntaxError*` the `synchronize` module would output this:
```
"msg": "SyntaxError parsing module.  Perhaps invoking \"python\" on your local (or delegate_to) machine invokes python3. You can set ansible_python_interpreter for localhost (or the delegate_to machine) to the location of python2 to fix this",
```

It's confusing because it started appearing after an upgrade but the return code of it was `0`.

This does not impact me that much because I ran tests well enough to not pay attention. I also already checked the return code which I trust. But definitively confusing and was also alarming when it started to happen in production :)

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

`synchronize` module

##### ANSIBLE VERSION

```
ansible 2.10.0.dev0
  config file = None
  configured module search path = [u'/home/creator/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.15+ (default, Oct  7 2019, 17:39:04) [GCC 7.4.0]
``` 

##### CONFIGURATION

```paste below

```

##### OS / ENVIRONMENT

```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.3 LTS"
```

##### STEPS TO REPRODUCE

```
mkdir /tmp/blah
touch /tmp/blah/SyntaxError.py
```

```
---
- hosts: localhost
  tasks:
    - synchronize:
        src: /tmp/blah/
        dest: /tmp/blih/
      delegate_to: localhost
```

##### ACTUAL RESULTS

`msg` should be `stdout_lines` with `\n`

``` 
"msg": "SyntaxError parsing module.  Perhaps invoking \"python\" on your local (or delegate_to) machine invokes python3. You can set ansible_python_interpreter for localhost (or the delegate_to machine) to the location of python2 to fix this",
    "rc": 0,
    "stdout_lines": [
        ".d..t...... ./",
        ">f+++++++++ SyntaxError.py"
    ]
```

I think the issue is there https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/action/synchronize.py#L420 Not sure how to help fix it.

### PR DESCRIPTION
**Pull request by medmrgh**
##### SUMMARY

Synchronize : remove unused block

Fixes #63954

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

synchronize

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/63954-synchronize-remove-unused-block.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- "Synchronize : Remove unused block "
```
Comment (samdoran) [inline changelogs/fragments/63954-synchronize-remove-unused-block.yml]:
```suggestion
- synchronize - remove unused block (https://github.com/ansible/ansible/issues/63954)
```

**Comment by medmrgh** [comment]
The CI/CD is keep failing due to external dependencies (mainly docker). Any help about this ?

**Comment by samdoran** [comment]
I re-ran the tests. The seems unrelated to this change. Hopefully it is just a transient failure.

**Comment by mscherer** [comment]
shipit

**Comment by samdoran** [comment]
Any reason this was closed? It had passing tests and was ready to be merged.

**Comment by pilou-** [comment]
@samdoran this PR was created during [the Ansible sprint at PyConFR](http://localhost:5000/2019/fr/talks/sprint.html#D%C3%A9veloppements%20de%20correctifs%20et%20tests%20des%20modules%20Ansible%20(&)), we will try to meet the author this week end.

**Comment by medmrgh** [comment]
My first PR/contribution has been closed/lost due to a hard sync from upstream. I recreated again the PR below : https://github.com/ansible/ansible/pull/64344

**Pull request closed without merging**